lib/node_modules/@stdlib/assert/is-same-date-object/README.md

```diff
@@ -0,0 +1,93 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# isSameDateObject
+
+> Test if two arguments are both Date objects and represent the same date object
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var isSameDateObject = require( '@stdlib/assert/is-same-date-object' );
+```
+
+#### isSameDateObject( d1 , d2 )
+
+Tests if two arguments are both valid date objects and both resolve to the same timestamp.
+
+```javascript
+var d1 = new Date(2024, 11, 31, 23, 59, 59, 999);
```

Comment:
@adityacodes30 Would you mind updating here and elsewhere to adhere to common project conventions? Namely, with some exception, we prefer spaces within brackets. E.g.,

```suggestion
var d1 = new Date( 2024, 11, 31, 23, 59, 59, 999 );
```

Reply:
will do that pronto